src/sage/combinat/symmetric_group_algebra.py

```diff
@@ -1941,6 +1941,45 @@ def _dft_seminormal(self, mult='l2r'):
         snb = self.seminormal_basis(mult=mult)
         return matrix([vector(b) for b in snb]).inverse().transpose()
 
+    def _dft_modular(self):
+        """
+        Return the discrete Foruier transform when the characteristic divides the order of the group.
+        See [Mur1983]_ for contrstruction of central primitive orthogonal idempotents.
+        For each idempotent e_i we have a projection `v \mapsto v*e_i`. This is a homomorphism.
```

Comment:
```suggestion
        For each idempotent e_i we have a projection `v \\mapsto v*e_i`. This is a homomorphism.
```
Alternatively you can avoid the double backslashes altogether by replacing line 1945 by `r"""`